### ISSUE TITLE
Deprecate this repo

### ISSUE THREAD

**Issue by ggwpez** [paritytech/smoldot#3112]
I think we should? Otherwise people will continue to open issues etc.

**Comment by ntn-x2** [comment]
Where is light client work happening then? In the main polkadot-sdk repo?

**Comment by ggwpez** [comment]
From the README: https://github.com/smol-dot/smoldot

**Comment by niklasp** [comment]
yes please i was thinking this is the official and the other one is fake before i found this issue
